00-framework-02-document-header.md

```diff
@@ -0,0 +1,157 @@
+|-----------------|--------------------------|--------|-----------------|
+| 2A              | Candidate Recommendation | Active | r0, 2019-05-27  |
+
+
+Authors: [@author1], [@author2]
+
+Interest Group: [@interested1], [@interested2]
+
+[@author1]: https://github.com/author1
+[@author2]: https://github.com/author2
+[@interested1]: https://github.com/interested1
+[@interested2]: https://github.com/interested2
+
+See the [lifecycle document][lifecycle-spec] for context about maturity level
+and spec status.
+
+[lifecycle-spec]: https://github.com/libp2p/specs/blob/master/00-framework-01-spec-lifecycle.md
+```
+
+## Format
+
+### Title and Short Intro
+
+Each spec begins with its title, formatted as an H1 markdown header. 
+
+The title can optionally be followed by a short block-quote introducing the
+spec, which should be a maximum of one or two lines.
+
+### Status Table
+
+The main status information is contained in a markdown table, using the [table
+syntax][gfm-tables] supported by [Github Flavored Markdown][gfm-spec].
+
+The status table consists of a single row, with a header containing the field
+names. 
+
+Example:
+
+```markdown
+| Lifecycle Stage | Maturity      | Status | Latest Revision |
+|-----------------|---------------|--------|-----------------|
+| 1A              | Working Draft | Active | r0, 2019-05-27  |
+```
+
+The following fields are all required:
+
+- `Lifecycle Stage`
+  - The [abbreviated lifecycle stage][abbrev-stage-definition] that the spec is
+    currently in. This must match the `Maturity` and `Status` fields.
+- `Maturity`
+  - The full name of the maturity level that the spec is currently in.
+  - Valid values are: `Working Draft`, `Candidate Recommenation`,
+    `Recommendation`
+- `Status`
+  - The full name of the status that the spec is currently in.
+  - For `Candidate Recommendation` or `Recommendation` specs, valid values are
+    `Active` and `Deprecated`
+  - For `Working Draft` specs, valid values are: `Active` and `Terminated`
+- `Latest Revision`
+  - A revision number and date to indicate when the spec was last modified,
+    separated by a comma.
+  - Revision numbers start with lowercase `r` followed by an integer, which gets
+    bumped whenever the spec is modified by merging a new PR.
+  - Revision numbers start at `r0` when the spec is first merged.
+  - Dates are formatted according to [ISO 8601](https://xkcd.com/1179/)
+
+### Authors and Interest Group
+
+After the status table, spec Authors and Interest Group members are listed.
+
+Authors and Interest Group members are referenced by their Github handles
+(with a leading `@` symbol), and are presented as a comma-separated list of links
+to Github profiles.
+
+To make the list readable in the markdown source, we use the [shortcut reference
+link syntax][gfm-shortcut-refs], which allows us to wrap the author name in
+square brackets in the list and define the link target below. For example:
```

Comment:
Albeit this may appear redundant when viewing in github.com with the GitHub renderer, it's necessary to avoid losing information when viewing elsewhere.